src/App.js

```diff
@@ -1,19 +1,135 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
 
+import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+const preparedTodos = todos.map(
+  todo => Object.assign(
+    todo, { user: users.filter(user => user.id === todo.userId)[0] },
+  ),
```

Comment:
```suggestion
  todo => ({
    ...todo,
    user: users.find(user => user.id === todo.userId) 
  });
```
зачем filter, если ты ищешь одну запись?

Reply:
done